tests/rules/test_key_duplicates.py

```diff
@@ -179,3 +179,44 @@ def test_key_tokens_in_flow_sequences(self):
                    '[\n'
                    '  flow: sequence, with, key: value, mappings\n'
                    ']\n', conf)
+
+    def test_forbid_duplicated_merge_keys(self):
+        conf = 'key-duplicates: {forbid-duplicated-merge-keys: true}'
+        self.check('---\n'
+                   'Multiple Merge Keys are NOT OK:\n'
+                   'anchor_one: &anchor_one\n'
+                   '  one: one\n'
+                   'anchor_two: &anchor_two\n'
+                   '  two: two\n'
+                   'anchor_reference:\n'
+                   '  <<: *anchor_one\n'
+                   '  <<: *anchor_two\n', conf, problem=(9, 3))
+        self.check('---\n'
+                   'Multiple Merge Keys are NOT OK:\n'
+                   'anchor_one: &anchor_one\n'
+                   '  one: one\n'
+                   'anchor_two: &anchor_two\n'
+                   '  two: two\n'
+                   'anchor_reference:\n'
+                   '  a: 1\n'
+                   '  <<: *anchor_one\n'
+                   '  b: 2\n'
+                   '  <<: *anchor_two\n', conf, problem=(11, 3))
```

Comment:
Can you add a case here to check when there are 3 `<<` merge markers? (I would expect 2 errors then)

Reply:
Added